perma_web/static/css/_perma-imports.scss

```diff
@@ -5,5 +5,4 @@
 @import "_fonts.scss";						// 2. import font definitions
 @import "_vars-global.scss";				// 3. import global variables
 @import "_mixins-global.scss";			// 4. import global mixins
-//@import "_placeholders-global.scss";		// 5. import global placeholders
-//@import "_reset.scss";						// 6. style reset and global defaults
\ No newline at end of file
+//@import "_placeholders-global.scss";		// 5. import global placeholders
```

Comment:
Curious why this is retained. Is it sometimes used, despite being commented out?